### PR TITLE
artist drive skeleton UI

### PR DESCRIPTION
**Pull request by adits31**
Screen Shot 2021-02-05 at 10 40 39 PM

Main TODOs left:
- Handle overflows properly
- Set max-widths on thumbnails
- Update Song model type
- Add helper to properly render uploaded at date.

Will take a pass at these in our next rev here.

### CODE REVIEW

File: page-components/ArtistDrive/SongThumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+import { Label3, Label4 } from 'components/Typography';
+import { Button } from 'components/Button';
+import type { SongThumbnailProps } from './types';
+import { formatSongLength } from 'utilities/timeUtils';
+
+
+const SongThumbnail: React.FC<SongThumbnailProps> = (props: SongThumbnailProps) => {
```
Comment (fezheng397) [inline page-components/ArtistDrive/SongThumbnail.tsx:14]: We normally just define the props types right above the component in the same file

File: page-components/ArtistDrive/mocks.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import type { Song } from 'types/model';
+
+export const mockSongList: Array<Song> = [
```
Comment (fezheng397) [inline page-components/ArtistDrive/mocks.ts:3]: i'll move this mock file to the /mocks/api file once this pr is merged

File: types/model.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// These are temporary for now - will be updated once we nail down the backend models
```
Comment (fezheng397) [inline types/model.ts:1]: we can probably just rename this to Song.ts and move it to the types/16bars folder like the other entity models here 
image

Comment (adits31) [reply types/model.ts:1]: Yeah that's fair - should be easy enough to move around later, can leave it like this for now.

File: page-components/ArtistDrive/mocks.ts (hunk 2 above)
Comment (adits31) [reply page-components/ArtistDrive/mocks.ts:3]: KK

**Review by fezheng397 — APPROVED** [review]

**Pull request merged**